customer-docs/user-manual.md

```diff
@@ -516,10 +519,11 @@ CLI Arguments:
 | output-path | String | True | Location where Streamliner output should be written to. |
 | type-mapping | String | True | Location of the type-mapping.yml file |
 | template-directory | String | True | Location of the templates |
+| log-level | String | False | Parameter to change the application log level |
 
 CMD : `<install directory>/bin/streamliner scripts --config conf/private-ingest-configuration.yml --state-directory <state-directory-path> --previous-state-directory <previous-state-directory-path> --type-mapping <install-directory>/conf/type-mapping.yml  --template-directory <install directory>/templates/<snowflake | hadoop> --output-path <script-output-path>`
```

Comment:
can we add the loge_level in the command?
If not added what is the default?

Reply:
updated the docs